city_scrapers/spiders/wayne_state_university.py

```diff
@@ -0,0 +1,94 @@
+from city_scrapers_core.constants import BOARD, COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+import re
+
+class WayneStateUniversitySpider(CityScrapersSpider):
+    name = "wayne_state_university"
+    agency = "Wayne State University"
+    timezone = "America/Chicago"
+    allowed_domains = ["http://bog.wayne.edu"]
+    start_urls = ["http://bog.wayne.edu/"]
+    location = {
+            "name": "McGregor Memorial Conference Center", 
+            "address": "Room BC 495 Gilmour Mall, Detroit, MI 48202",
+        }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        self._validate_location(response)
+        for item in response.xpath('//*[@id="content"]/div[1]/div/h2').re('((?:January|Feburary|March|April|May|June|July|August|October|September|November|December)\s+\d{1,2},\s+\d{4})'):
+            meetings = self._parse_meeting_types(response)
+            for meeting_type in meetings: 
+                meeting = Meeting(
+                    title = meeting_type[0],
+                    description=self._parse_description(item),
+                    classification= meeting_type[-1], 
+                    start= meeting_type[1],
+                    end= "Ending Time Not Specified",
+                    all_day=self._parse_all_day(item),
+                    time_notes=self._parse_time_notes(item),
+                    location=self.location,
+                    links=self._parse_links(item,response),
+                    source=self._parse_source(response),
+                )
+                yield meeting
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+            
+
+    def _parse_meeting_types(self, response):
+        """ Parse meeting titles and start times. Returns a list of tuples including titles and meeting times """
+        meeting_details = []
+        meeting_types = []
+        for item in response.css('#content .border-t li').extract(): #interate through meeting times and types
+            start_time = re.search('(\d+:\d{1,2}\s(?:a.m|p.m))', item)
```

Comment:
`start` needs to be a `datetime` object, so you'll need to parse any string you find